### PR TITLE
Rewritting realexp()

### PR DESCRIPTION
**Pull request by choisy**
Addressing issue #100

### CODE REVIEW

File: R/run_experiment.R [hunk 1]
```diff
@@ -11,40 +11,17 @@
...
-    else {
-      newoutput <- c(newoutput, list(curoutput))
-    }
-  }
-  return(newoutput)
+realexp <- function(output, exp) {
+  op <- obs_rates(exp)
+  mapply(function(nbrow, obsper, df) {
+           xs <- lapply(obsper, function(by) setdiff(1:nbrow, seq(1, nbrow, by)))
+           ys <- sapply(names(indexes), grep, names(df))
```
Comment (LucieContamin) [inline R/run_experiment.R]: The new version is not working because the object `indexes` does not exist.

Comment (choisy) [reply R/run_experiment.R]: Ooopsss... Bug fixed now.

Comment (LucieContamin) [reply R/run_experiment.R]: Perfect! Thank you

**Comment by LucieContamin** [comment]
Thank you, the new version seems to work perfectly!

**Pull request merged**